lib/screens/play/view_active_quests_page.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:treasure_of_the_high_seas/model/game_state.dart';
+import 'card_viewer.dart';
+
+class ViewActiveQuestsPage extends StatelessWidget {
+  final GameState state;
+
+  const ViewActiveQuestsPage(this.state);
+  @override
+  Widget build(BuildContext context) {
+    final activeQuests = state.getActiveQuestCards();
+
+    return Scaffold(
+        appBar: AppBar(
+          title: Text('Active Quests'),
+        ),
+        body: ChangeNotifierProvider.value(
+            value: state,
+            child: Consumer<GameState>(builder: (context, state, _) {
+              return Column(children: <Widget>[
+                Expanded(child: CardViewer(activeQuests, buttons)),
+                Container(
+                    padding: EdgeInsets.symmetric(vertical: 5),
+                    width: MediaQuery.of(context).size.width * 0.7,
+                    child: RaisedButton(
+                      shape: RoundedRectangleBorder(
+                        side: BorderSide(color: Colors.black, width: 1),
+                        borderRadius: BorderRadius.circular(10),
+                      ),
+                      onPressed: () {
+                        Navigator.of(context).pop();
+                      },
+                      color: Colors.green,
+                      child: Text("Close", style: TextStyle(fontSize: 30)),
+                    ))
+              ]);
+            })));
```

Comment:
There are some opportunities for code reuse here, e.g.

1) Extract out a `ReadOnlyCardViewer` which takes a list of cards and renderes the CardViewer and close buttons. Then this page and the view discard one become a couple of lines.
2) Just extract the close button into some sort of `widget_utils`, although that doesn't feel as valuable.

Idk, up to you - maybe we apply the rule of three and hold off for now. It would make things more of a pain down the road if these pages were to diverge, but also DRY :shrug:

Reply:
When I discussed approaches to this with Mark, he suggested "Would be a new button in the top bar and maybe a swipeable thing like the discard pile? Or into a new page that's a list of card names, but only displaying ones in the deck or discard that have quest type"

So I wasn't 100% sure that "do it like the discard pile" was the best approach and extracting the common features seemed premature. I guess whether we do factor out ReadOnlyCardViewer depends on whether the discard pile and quests view are likely to cohere with one another long-term?

I agree that extracting out the close button feels less worthwhile.

This maybe also ties into my suggestion in the PR description that the read-only card viewers shouldn't show the "Discard" and "Exile" options on the cards, as I don't think it makes sense to include them in a read-only context, right? Maybe we could raise an issue to do that, and look at refactoring as part of the same task?

Reply:
I quite like the discard pile-like approach, because it means that as well as seeing what the current quests are you can also see their requirements, which lets you plan ahead in terms of your resources a bit. 

I'm not sure I agree on the "Discard" / "Exile" options front - or perhaps I'm misunderstanding what you're suggesting. But to my mind you want to see the full details of the card, which includes what actions are available and what they would do if chosen during gameplay?

Reply:
Yes, good point, I hadn't considered the usefulness of seeing those options ahead of time. My point was that the options on those cards which are tappable during gameplay are not tappable when displayed in the viewing page, but appear to be displayed in exactly the same way. Therefore, maybe they could be greyed out or something? Or, maybe it's not nearly as likely to be confusing as I am speculating it might be. I guess it's even less likely to be confusing in the discard view, since the cards have obviously been discarded.  I thought the active quests might be a different story though.

Reply:
Also, I raised https://github.com/kilikfan/treasure/issues/90 for adding card_viewer tests further to your original comment.

Reply:
Sorry for the confusion above, it turns out I was misunderstanding how the game works - I thought the destinations on each path were separately tappable options. Mark has set me straight. (I did somehow manage to win a game last night operating on that misunderstanding though!)

I'm fully prepared to accept that it might just be me, but I'm wondering what your opinion is on the wording of the rules currently - I'm thinking that it might be that it could be made less prone to being misunderstood by someone who isn't already familiar with the game.